config/config.go

```diff
@@ -41,7 +58,9 @@ func Get() (*Config, error) {
 		MailPassword:               "",
 		FeedbackTo:                 "to@gmail.com",
 		FeedbackFrom:               "from@gmail.com",
-		RendererURL:                "http://localhost:20010",
+		SiteDomain:                 "localhost",
```

Comment:
Have we updated the secrets for prod and sandbox?

Reply:
i haven't as part of this task. assumed it would be another task?

Reply:
Cool that is fine, just wanted to check and I'll create some tickets to do this separately